### ISSUE TITLE
Unable to run the project with IntelliJ IDEA when the user name contains a space

### ISSUE THREAD

**Issue by salmonb** [codenameone/CodenameOne#1610]
The CN1 plugin generates the following (truncated) command line which fails (I replaced the user name with *Firstname Lastname* to make things clearer):

```
"C:\Program Files\Java\jdk1.8.0_60\bin\java" -server -Xms128m -Xmx512m -XX:MaxPermSize=250m -XX:ReservedCodeCacheSize=240m -XX:+UseConcMarkSweepGC -XX:SoftRefLRUPolicyMSPerMB=50 -ea -Dsun.io.useCanonCaches=false -Djava.net.preferIPv4Stack=true -XX:+HeapDumpOnOutOfMemoryError -XX:-OmitStackTraceInFastThrow -Duser.name=Firstname Lastname etc...
```

It complains about not finding the class *Lastname*. I guess this is because there is a space in the user name and -Duser.name=Firstname Lastname breaks the arguments chain.

**Comment by codenameone** [comment]
I have no idea where that comes from? 
I think that's IntelliJ code not ours.

**Comment by salmonb** [comment]
I checked and you are right. Althought this command line is triggered by the CN1 plugin when I run the app, all the VM arguments are added by IntelliJ from the idea.exe.vmoptions file (which includes -Duser.name=Firstname Lastname in my case). So I opened the [issue on stackoverflow](http://stackoverflow.com/questions/33956925/how-to-change-the-default-author-namein-intellij-idea-without-side-effects) and I'm now closing this issue here. Sorry and thanks.
